tests/dash/packets.py

```diff
@@ -103,13 +103,13 @@ def inbound_pl_packets(config, inner_packet_type='udp', vxlan_udp_dport=4789):
         ip_src=pl.PE_PA,
         ip_dst=pl.APPLIANCE_VIP,
         gre_key_present=True,
-        gre_key=int(pl.VNET1_VNI) << 8,
+        gre_key=int(pl.ENCAP_VNI) << 8 if hwsku == "Cisco-8102-28FH-DPU-O-T1" else int(pl.VNET1_VNI) << 8,
```

Comment:
can we check the HWSKU outside of this function, maybe in the test function itself, and just have a parameter to decide which VNI to use? IMO packet generation should be ignorant of HWSKU concerns, and we may need to update the affected SKUs later anyways